### ISSUE TITLE
I am not able to create KMS - TFv 1.1.0

### ISSUE THREAD

**Issue by pysiok1983** [apigee/terraform-modules#22]
```
│ Error: Inconsistent conditional result types
│
│   on .terraform\modules\kms-org-db\modules\kms\main.tf line 53, in locals:
│   52:     var.keyring_create
│   53:     ? google_kms_key_ring.default.0
│   54:     : data.google_kms_key_ring.default.0
│     ├────────────────
│     │ data.google_kms_key_ring.default[0] is a object, known only after apply
│     │ google_kms_key_ring.default[0] is a object, known only after apply
│     │ var.keyring_create is a bool, known only after apply
│
│ The true and false result expressions must have consistent types. The given expressions are object and object, respectively.
```

**Comment by danistrebel** [comment]
@pysiok1983 I am running with tf version 1.1.7 and I didn't observe any issues. Are you able to upgrade tf and try again?

**Comment by danistrebel** [comment]
closing this in favor of the question in the upstream repo
